docker/docker-compose.yml

```diff
@@ -38,6 +38,8 @@ services:
     volumes:
       - /var/run/aesmd:/var/run/aesmd
       - /var/run/ehsm:/var/run/ehsm
+      - /var/local/ehsm:/var/local/ehsm

```

Comment:
can we use environments directly rather than hardcode?

Reply:
Certainly, the latest commit implemented this, please check.